awswrangler/pandas/__init__.py

```diff
@@ -0,0 +1,52 @@
+"""Pandas "proxy" package."""
+
+from awswrangler._distributed import MemoryFormatEnum, memory_format
+
+if memory_format.get() == MemoryFormatEnum.PANDAS:
```

Comment:
When I ask MyPy to evaluate what the return type is for functions such as `wr.dynamodb.read_items` , it tells me the function will return a Modin DataFrame. If I uninstall `modin` from my environment, then it's unable to evaluate that `pd.DataFrame` is meant to be a `pandas` DataFrame.

I'm not sure why exactly it behaves this way, but it seems MyPy is already struggling to evaluate types from this proxy package.

Between Modin and Pandas, I think it would be best if the type checker just defaulted to thinking its always returning pandas DataFrames for now. This can easily be achieved by changing this line to:

```python
if TYPE_CHECKING or memory_format.get() == MemoryFormatEnum.PANDAS:
```

Reply:
Yeah I agree. I will look into the types in the next PRs, for now tbh just wanted to give this "proxy" module idea a shot